src/main/resources/public/js/index.js

```diff
@@ -155,6 +178,29 @@ const store = Vuex.createStore({
           }
         }
       }
+      state.consolidatedIngredients.sort(
+        (a, b) => {
```

Comment:
This comparator is exactly the same as the other one. This should be a named function (say `ingComparator`) and then you call sort like this: `state.consolidatedIngredients.sort(ingComparator);`